types/applepayjs/index.d.ts

```diff
@@ -1050,6 +1094,16 @@ declare namespace ApplePayJS {
         readonly validationURL: string;
     }
 
+    /**
+     * The attributes contained by the oncouponcodechanged callback function.
+     */
+    abstract class ApplePayCouponCodeChangedEvent extends Event {
+        /**
+         * The coupon code entered by the user.
```

Comment:
```suggestion
         * The updated coupon code from the payment sheet.
```

Reply:
Done, thanks! Could you please take a look?